CHANGELOG.md

```diff
@@ -50,6 +55,7 @@ This Splunk OpenTelemetry Connector release includes changes from the [opentelem
 
 - [`httpsink` exporter](https://github.com/signalfx/splunk-otel-collector/tree/main/internal/exporter/httpsinkexporter) to make span data available via a HTTP endpoint
 - Initial release of [`translatesfx`](https://github.com/signalfx/splunk-otel-collector/tree/main/cmd/translatesfx) to translate a SignalFx Smart Agent configuration file into a configuration that can be used by an OpenTelemetry Collector
+- Add Chocolately package support. (#724)
```

Comment:
This is in main but hasn't actually been released yet.

Reply:
oh, I assumed it was an oversight